src/Security/Authorization/Core/src/IAuthorizationPolicyProvider.cs

```diff
@@ -22,5 +22,11 @@ public interface IAuthorizationPolicyProvider
         /// </summary>
         /// <returns>The default authorization policy.</returns>
         Task<AuthorizationPolicy> GetDefaultPolicyAsync();
+
+        /// <summary>
+        /// Gets the fallback authorization policy.
+        /// </summary>
+        /// <returns>The fallback authorization policy.</returns>
+        Task<AuthorizationPolicy> GetFallbackPolicyAsync() => GetDefaultPolicyAsync();
```

Comment:
Returning null would be a better default. FallbackPolicy defaults to null where DefaultPolicy requires auth.

Reply:
Yeah good catch, I'll add a test for the default behavior too